sections/01_abstract.md

```diff
@@ -3,3 +3,14 @@
 Offspring of the [Deep Review](https://github.com/greenelab/deep-review).
 We cite the Deep Review like [@doi:10.1101/142760] or [@tag:deep_review].
 The source repository is [@url:https://github.com/greenelab/manubot-rootstock].
+
+Equations can be integrated and labeled like this `{{ '{{ \'{#eq:label}\' }}' }}` and then
+referenced using `{@eq:label}` like this for Equation @eq:line.
+
+$$ y = mx + b $$ {{ '{#eq:line}' }}
+
+Images can be included and labeled like `{{ '{{ \'{#fig:label}\' }}' }}` and then
+referenced using `{@fig:label}` like this for Figure @fig:lotus.
```

Comment:
Regarding `{@fig:label}`: If [my reading](https://github.com/tomduck/pandoc-fignos#markdown-syntax) is correct, the brackets around image references is not needed. Let's not use them?